best-practices/documentation.md

```diff
@@ -0,0 +1,10 @@
+# Documentation
+
+## Ruby
+Ruby projects should use [YARD](http://yardoc.org/) for documentation purposes.
+
+[Getting Started with YARD](http://www.rubydoc.info/gems/yard/file/docs/GettingStarted.md)
+
+One benefit is YARD runs a live documentation server for the community, hosting docs for all RubyGems and Github projects. The project is at [RubyDoc.info](http://www.rubydoc.info/) (formerly rdoc.info)
+
+A [useful Yardoc cheatsheet](https://gist.github.com/chetan/1827484)
```

Comment:
Probably a good idea, since if something changes upstream in YARD we can own which changes we want to make (or not).